npm-packages/meteor-installer/README.md

```diff
@@ -1,16 +1,54 @@
 ## Meteor Installer
 
-Node.js <=14.x and npm <=6.x is recommended.
+### Recommended Versions
 
-Install Meteor by running:
+- For Meteor 2
+  - Use Node.js 14.x or higher
+  - Use npm 6.x or higher
+- For Meteor 3
+  - Use Node.js 20.x or higher
+  - Use npm 9.x or higher
+
+### Installation
+
+To install Meteor, run the following command:
 
 ```bash
 npm install -g meteor
 ```
 
-[Read more](https://www.meteor.com/developers/install)
+#### Running with `npx`:
+
+```bash
+npx meteor
+
+# or with a specific version
+npx meteor@<version>
+```
+
+This command will execute the Meteor installer without adding it permanently to your global npm packages.
+
+For more information, visit the [official Meteor installation guide](https://www.meteor.com/developers/install).
```

Comment:
This link goes to the Meteor 2 installer. Maybe we should add both links here?

https://v2-docs.meteor.com/install.html

https://v3-docs.meteor.com/about/install.html

Reply:
Yes, that makes sense. Thanks!